script.js

```diff
@@ -0,0 +1,18 @@
+window.onload = function() {
+
+const synth = new Tone.PolySynth(2, Tone.Synth, {
+    oscillator : {
+          type : "square"
+      }
+  }).toMaster();
+   
+document.querySelector('tone-keyboard').addEventListener('noteon', e => {
```

Comment:
I don't believe addEventListener has a type 'noteon'. Is this part of the tonejs UI library that you included?